srcs/vm/check_args.c

```diff
@@ -98,6 +99,33 @@ static void	loop_args(int argc, char **argv, t_arena *arena)
 	}
 }
 
+static void	set_champions(t_arena *arena, char **argv)
+{
+	return; // NOTE: WHAT DOES THIS DO AND WHY DOES IT WORK WITHOUT?
```

Comment:
@JoorsB I commented this out, still works perfectly fine? Why? What should this do?

Reply:
This sets the champions in order from the first arena->champions[index];

Without this, there is a possibility with the use of "-n" that there is an empty arena->champions[index] in between used ones.    *** This was the bug with use of "-n". ***